Example/watchOSSample/Podfile

```diff
@@ -1,10 +1,12 @@
+source 'https://github.com/firebase/SpecsStaging.git'
```

Comment:
Is this temporary for now or should it stay forever?

Reply:
It should stay forever to support GoogleUtilities integration testing